### DISCUSSION TITLE
[A-L] (2023/24) Foglio 17 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#383] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/1ca2d2fd-5a73-444c-ad6f-1c4315d7854c)

**Comment by Elia-Belli** [comment, marked as answer]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/3ace636f-04f6-4677-99ee-f8187403d740)

![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4c30abd4-7901-4540-8707-bfcafd98fc5a)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/ef1fd098-62d9-4e58-9285-f4455e13693a)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/eb8d3c64-0f26-4e16-8d53-a5a77becfb8c)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/aad86550-2204-4edc-9b69-62df012942fe)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/a9c707e2-59df-4082-b53a-75f9ddc4a476)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/cd60c9c2-ec54-48a9-90de-ddedb13406c3)

**Comment by CasuFrost** [comment]
[22 dicembre SVOLTO.pdf](https://github.com/sapienzastudentsnetwork/algebra2324/files/13822071/22.dicembre.SVOLTO.pdf)
